app/src/actions/notifications.js

```diff
@@ -13,7 +13,7 @@ export const getNotifications = async dispatch => {
     }
     dispatch({
       type: types.SET_NOTIFICATIONS,
-      payload: response.data.items,
+      payload: { items: response.data.items, unreaded: response.data.unreaded },
```

Comment:
```suggestion
      payload: response.data,
```
Hace la misma funcion no?

Reply:
Si si, hace lo mismo que antes, ademas agrega un valor nuevo al reducer, q es "unreaded"